app/code/Magento/Cron/view/adminhtml/ui_component/adminhtml_cron_listing.xml

```diff
@@ -0,0 +1,437 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Try to rewrite configuration on new style(e.g https://github.com/magento/magento2/blob/develop/app/code/Magento/Customer/view/adminhtml/ui_component/customer_listing.xml).